src/test/java/org/folio/service/EcsTlrServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package org.folio.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.UUID;
+
+import org.folio.domain.dto.EcsTlr;
+import org.folio.domain.entity.EcsTlrEntity;
+import org.folio.domain.mapper.EcsTlrMapper;
+import org.folio.repository.EcsTlrRepository;
+import org.folio.service.impl.EcsTlrServiceImpl;
+import org.joda.time.DateTime;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class EcsTlrServiceTest {
+
+  @InjectMocks
+  private EcsTlrServiceImpl ecsTlrService;
+  @Mock
+  private EcsTlrRepository ecsTlrRepository;
+  @Mock
+  private EcsTlrMapper ecsTlrMapper;
```

Comment:
Do we have to mock the mapper here? Can we use a real one?

Reply:
updated, thanks